core/platform.js

```diff
@@ -10,8 +10,7 @@
 *        Copyright (c) Matthew Knox and Contributors 2015.
 */
 
-var figlet = require('figlet'),
-    shim = require.once('./shim.js'),
+var figlet = require('figlet');
 
 Platform = function(modes) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["Platform" is not defined.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530333306/issues/source?bid=3282987&fileBranchId=3282997#l15)